static/js/taxpayer_creation.js

```diff
@@ -4,9 +4,15 @@ var left, opacity, scale;
 var animating; 
 
 
-
+function check_complete(esto){
```

Comment:
Maybe we can change the variable name to something more representative

Reply:
that function was unused so i will remove it completely